content/pt-br/docs/reference/access-authn-authz/authorization.md

```diff
@@ -0,0 +1,248 @@
+   * Quando especificado RBAC (Role-Based Access Control) usa o group de API `rbac.authorization.k8s.io` para orientar as decisões de autorização, permitindo que os administradores configurem dinamicamente as políticas de permissão por meio da API do Kubernetes.
+   * Para habilitar o modo RBAC, inicie o servidor de API (apiserver) com a opção `--authorization-mode=RBAC`.
+ * **Webhook** - Um WebHook é um retorno de chamada HTTP: um HTTP POST que ocorre quando algo acontece; uma simples notificação de evento via HTTP POST. Um aplicativo da Web que implementa WebHooks postará uma mensagem em um URL quando ocorrerem determinadas coisas. Para saber mais sobre como usar o modo Webhook, consulte [Webhook Mode](/docs/reference/access-authn-authz/webhook/).
+
+#### Verificando acesso a API
+
+`kubectl` fornece o subcomando `auth can-i` para consultar rapidamente a camada de autorização da API.
+O comando usa a API `SelfSubjectAccessReview` para determinar se o usuário atual pode executar
+uma determinada ação e funciona independentemente do modo de autorização utilizado.
+
+
+```bash
+# "can-i create" = "posso criar"
+kubectl auth can-i create deployments --namespace dev
+```
+
+A saída é semelhante a esta:
+
+```
+yes
+```
+
+```shell
+# "can-i create" = "posso criar"
+kubectl auth can-i create deployments --namespace prod
+```
+
+A saída é semelhante a esta:
+
+```
+no
+```
+
+Os administradores podem combinar isso com [user impersonation](/pt-br/docs/reference/access-authn-authz/authentication/#user-impersonation)
+para determinar qual ação outros usuários podem executar.
+
+```bash
+# "can-i list" = "posso listar"
+
+kubectl auth can-i list secrets --namespace dev --as dave
+```
+
+A saída é semelhante a esta:
+
+```
+no
+```
+
+Da mesma forma, para verificar se uma ServiceAccount chamada `dev-sa` no Namespace `dev`
+pode listar Pods no namespace `target`:
+
+```bash
+# "can-i list" = "posso listar"
+kubectl auth can-i list pods \
+	--namespace target \
+	--as system:serviceaccount:dev:dev-sa
+```
+
+A saída é semelhante a esta:
+
+```
+yes
+```
+
+`SelfSubjectAccessReview` faz parte do grupo de API `authorization.k8s.io`, que
+expõe a autorização do servidor de API para serviços externos. Outros recursos em
+este grupo inclui:
+
+* `SubjectAccessReview` - * `SubjectAccessReview` - Revisão de acesso para qualquer usuário, não apenas o atual. Útil para delegar decisões de autorização para o servidor de API. Por exemplo, o kubelet e extensões de servidores de API utilizam disso para determinar o acesso do usuário às suas próprias APIs.
+
+* `LocalSubjectAccessReview` - Similar a `SubjectAccessReview`, mas restrito a um namespace específico.
+
+* `SelfSubjectRulesReview` - Uma revisão que retorna o conjunto de ações que um usuário pode executar em um namespace. Útil para usuários resumirem rapidamente seu próprio acesso ou para Interfaces de Usuário ocultarem/mostrar ações.
```

Comment:
Suggestion: remove capital initials from "Interfaces de Usuário".
Minor: "mostrarem" would be more correct as it refers to "interfaces de usuário".